core/tests/performance_framework/perf_services.py

```diff
@@ -200,7 +208,7 @@ def _setup_proxy_server(self, downstream_kbps=None, upstream_kbps=None,
 
         return server, proxy
 
-    def _setup_driver(self, proxy=None, use_proxy=False):
+    def setup_driver(self, proxy=None, use_proxy=False):
```

Comment:
Why is this public? It is used in a method in base.py, but that method is never used. If you remove that method, it seems like this can stay private.